src/github/createPRViewProviderNew.ts

```diff
@@ -158,6 +159,24 @@ export class CreatePullRequestViewProviderNew extends WebviewViewBase implements
 		return undefined;
 	}
 
+	private async getPullRequestDefaultLabels(defaultBaseRemote: RemoteInfo): Promise<ILabel[]> {
+
+		const defaultLabelValues = vscode.workspace.getConfiguration(PR_SETTINGS_NAMESPACE).get(PULL_REQUEST_LABELS) as Array<string>;
```

Comment:
```suggestion
		const defaultLabelValues = vscode.workspace.getConfiguration(PR_SETTINGS_NAMESPACE).get<string[]>(PULL_REQUEST_LABELS);
```

Reply:
Also, it would be cool if we merged the user settings and the workspace settings. You can get the user and workspace settings by useing `.inspect` instead of `.get`.

Reply:
Done!

<img width="1624" alt="Screenshot 2024-02-04 at 7 25 27 PM" src="https://github.com/microsoft/vscode-pull-request-github/assets/41837826/4d5ddb17-fcef-4d20-8c79-ec9354a7730a">